packages/quantic/force-app/main/default/lwc/quanticColoredResultBadge/quanticColoredResultBadge.js

```diff
@@ -20,6 +19,23 @@ export default class QuanticColoredResultBadge extends LightningElement {
    * @type {string}
    */
   @api label;
```

Comment:
The comment could specify that if `label` is specified, then `field` will be ignored?